migration/mule/rest-api-with-apikit/RestApiWithApikitCompositeApplication/.project

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>RestApiWithApikitCompositeApplication</name>
+	<comment></comment>
+	<projects>
+	</projects>
+	<buildSpec>
+	</buildSpec>
+	<natures>
+		<nature>org.wso2.developerstudio.eclipse.distribution.project.nature</nature>
```

Comment:
shall we check the indentation here? seems a bit inconsistent

Reply:
These files were removed from the repo after a restructuring, hence will not be able to do this